rjc_account_form/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': 'RJC account report and preprint',
+    'version': '12.0.1.0.0',
+    'author': 'Ecosoft',
+    'license': 'AGPL-3',
+    'website': 'https://github.com/ecosoft-odoo/rjc',
+    'category': 'Report',
+    'depends': ['account',
```

Comment:
```suggestion
    'depends': [
        'account',
```
one line, if one depend.